test/integration/policy_scc_test.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the policy-scc policy", Label("policy-collection", "stable"), func() {
+
+	const (
+		rootPolicyName   = "policy-scc"
+		rootPolicyURL    = policyCollectSCURL + rootPolicyName + ".yaml"
+		rootPolicyNSName = "policy-scc"
+		targetName       = "restricted"
+		targetKind       = "scc"
+	)
+
+	var (
+		targetGVR = common.GvrSCC
+	)
+
+	It("stable/"+rootPolicyName+" should be created on the Hub", func() {
+		By("Creating the policy on the Hub")
+		_, err := utils.KubectlWithOutput(
+			"apply", "-f", rootPolicyURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Creating the " + rootPolicyNSName + " namespace on the managed cluster")
+		namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: rootPolicyNSName, Labels: map[string]string{"e2e": "true"}}}
+		_, err = clientManaged.CoreV1().Namespaces().Create(context.TODO(), namespace, metav1.CreateOptions{})
+		Expect(err).To(BeNil())
+
+		By("Patching the namespaceSelector to use the " + rootPolicyNSName + " namespace")
+		_, err = clientHubDynamic.Resource(common.GvrPolicy).Namespace(userNamespace).Patch(
+			context.TODO(),
+			rootPolicyName,
+			k8stypes.JSONPatchType,
+			[]byte(`[{"op": "replace", "path": "/spec/policy-templates/0/objectDefinition/spec/namespaceSelector/include", "value": ["`+rootPolicyNSName+`"]}]`),
+			metav1.PatchOptions{},
+		)
+		Expect(err).To(BeNil())
+
+		By("Patching placement rule")
+		err = common.PatchPlacementRule(
+			userNamespace, "placement-"+rootPolicyName, clusterNamespace, kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Checking that " + rootPolicyName + " exists on the Hub cluster")
+		rootPolicy := utils.GetWithTimeout(
+			clientHubDynamic, common.GvrPolicy, rootPolicyName, userNamespace, true, defaultTimeoutSeconds,
```

Comment:
policy name is `policy-securitycontextconstraints`

Reply:
file name is `policy-scc`

Reply:
Oh, man! Fixing now. Looks like SCC isn't namespaced, either. And this particular one might be preinstalled on clusters, so it's probably always compliant...